tests/resources/Util.robot

```diff
@@ -123,6 +128,16 @@ Retry Wait Until Page Not Contains Element
     @{param}  Create List  ${element_xpath}
     Retry Action Keyword  Wait Until Page Does Not Contain Element  @{param}
 
+Retry Wait Until Keyword Succeeds
+    [Arguments]  ${element_xpath}
+    @{param}  Create List  ${element_xpath}
+    Retry Action Keyword  Wait Until Keyword Succeeds  @{param}
+
+
+Link Click
```

Comment:
do we really need this keyword?

Reply:
It's the same with other keywords, add a retry to make sure element can be clicked.